usefulUtilities/sit/sitClient.js

```diff
@@ -446,57 +439,66 @@
         canClickOnModelToSit: false
     };
```

Comment:
missing empty line between functions

Reply:
I think what's here is correct; `DEFAULT_SIT_USER_DATA_WITH_CUSTOM_SETTINGS` is not a function